src/js/model/getWinnerNames.js

```diff
@@ -0,0 +1,9 @@
+import getMaxForwardPlayerName from '../ui/getMaxForwardPlayerName.js';
+import getMaxScore from '../ui/getMaxScore.js';
+
+export default function getWinnerNames() {
+  const maxScore = getMaxScore();
```

Comment:
```suggestion
export default function getWinnerNames(maxScore) {
```
이렇게 함수를 실행할 때 인자로 받아와야 하지 않을까요?
getMaxScore를 model에서 호출하게 되면 결국 dom에 대한 종속이 있는거라서요!